Library/Homebrew/compat/md5.rb

```diff
@@ -1,26 +1,35 @@
 class Formula
   def self.md5(val)
+    # Don't throw an error at load time, just install time.
     stable.md5(val)
   end
 end
 
 class SoftwareSpec
   def md5(val)
+    odie <<-EOS.undent
+      Using MD5 has been dropped for security reasons.
+      Please switch to SHA-256.
+    EOS
```

Comment:
> For MD5 we want to kill brew.

I disagree. Noted we are talking load time(i.e. any brew command that load formulae) not run time (like brew install). Therefore, to users, it would be appear that some random commands will cause brew exit simply because one bad formula. 

What makes it worse is that `odie` unlike `raise RuntimeError` won't be properly caught. One of examples is `brew tap` will just exit without untap the tap if one of formulae happens to use such DSL.